staging/src/k8s.io/client-go/tools/auth/exec/types_test.go

```diff
@@ -17,26 +17,39 @@ limitations under the License.
 package exec
 
 import (
+	"fmt"
 	"reflect"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	clientauthenticationv1 "k8s.io/client-go/pkg/apis/clientauthentication/v1"
 	clientauthenticationv1alpha1 "k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1"
 	clientauthenticationv1beta1 "k8s.io/client-go/pkg/apis/clientauthentication/v1beta1"
 	clientcmdv1 "k8s.io/client-go/tools/clientcmd/api/v1"
 )
 
-// TestV1beta1ClusterTypesAreSynced ensures that clientauthenticationv1beta1.Cluster stays in sync
+func TestClientAuthenticationClusterTypesAreSynced(t *testing.T) {
+	t.Parallel()
+
+	for _, cluster := range []interface{}{
+		clientauthenticationv1beta1.Cluster{},
+		clientauthenticationv1.Cluster{},
+	} {
+		t.Run(fmt.Sprintf("%T", cluster), func(t *testing.T) {
+			testClientAuthenticationClusterTypesAreSynced(t, cluster)
```

Comment:
You are missing a `t.Parallel()` here.

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/86ba7ca7f21e0d8edaea14a81aba1dd5671c7535